packages/terminal/src/index.ts

```diff
@@ -25,3 +24,88 @@ export const ITerminalTracker = new Token<ITerminalTracker>(
   '@jupyterlab/terminal:ITerminalTracker'
 );
 /* tslint:enable */
+
+/**
+ * The namespace for terminals.
+ */
+export namespace ITerminal {
+  export interface ITerminal extends Widget {
+    /**
```

Comment:
It doesn't specify a constructor, but that's ok?

Reply:
Yeah, I think it's probably a mistake to specify constructors in interfaces, no?

Reply:
Oh, indeed. that's kinda weird, frankly, but maybe i'm not so type smart, i guess...

Reply:
It certainly makes it harder to use interfaces within a dependency injection context.